### ISSUE TITLE
What about case sensitivity?

### ISSUE THREAD

**Issue by Richomuffin** [avastmick/cryptographon#7]
The example clip shows the message coming back in lower case, when it was encoded with Sentence Case?

**Comment by avastmick** [comment]
This is correct, and intentional. The `cipher-one` code does not retain case in the encode/decode cycle. Please note, many classical ciphers are case insensitive.

The objective of the repo is to walk through from a starting point of a simple substitution cipher (as created by a 9-year-old) through a number of examples that fix issues with the implementation. `cipher-one` has key limitations as per the original cipher, these are alleviated in `cipher-two`, and further still in `cipher-three`. The latter fixes the case sensitivity issue.

Hope that makes sense.

If you are interested in contributing, that would be great. Documentation or code appreciated.

**Comment by Richomuffin** [comment]
I can't even wind you up anymore. You have forgotten so much from your time
in Wellington. ☺ I was just wondering how life was treating you in Shanghai
and if you had finished the book.
Nice idea with the open source literature btw.  Quite smart!

Rich.

**Comment by avastmick** [comment]
Ha ha! Right, I know two Richs in Welly, both equally capable of sucking me into the same (elaborate) gag. I'll put chips on Bremford, because cat.
